PlayerControls/sources/AdVideoControls.swift

```diff
@@ -23,10 +23,13 @@ public final class AdVideoControls: UIViewController {
     
     @IBOutlet public weak var containerView: UIView!
     
+    @IBOutlet public weak var airplayActiveLabel: UILabel!
```

Comment:
If airplays is active we will show the on top of advertisement?

Reply:
What do you mean "on top of advertisement"? Image casts to Apple TV, iPhone's screen is black and view will show on top of it.